projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/modifiers/remove_modifiers.json

```
@@ -7,7 +7,7 @@
   },
   "text": [
     {
-      "text": "“我发现用湿海绵可以去除工具上的强化。去除原有的强化会留下增加新强化的空间，有些用于强化的物品也可以回收。如果擦洗工具的位置和所用海绵的数量不同，能够去除的强化效果也有所不同。"
+      "text": "“我发现用湿海绵可以去除工具上的强化。去除原有的强化会留下增加新强化的空间，有些用于强化的物品也可以回收。擦洗工具的位置和所用海绵的数量不同，那么能够去除的强化效果也有所不同。"
```

```suggestion
      "text": "“我发现用湿海绵可以去除工具上的强化。去除原有的强化会留下增加新强化的空间，有些用于强化的物品也可以回收。擦洗工具的位置和所用海绵的数量不同，能够去除的强化效果也有所不同。"
```
我觉得要么是“如果……那么……”，要么是像我改的这样啥都不加吧。前面没有“如果”的时候后面又有一个“那么”读起来怪怪的，不是很通顺。